packages/ses/src/compartment-shim.js

```diff
@@ -212,6 +381,14 @@ export const makeCompartmentConstructor = (
       // The snapshot is frozen to ensure that the properties are immutable
       // when transferred-by-property-descriptor onto local scope objects.
       globalLexicals: freeze({ ...globalLexicals }),
```

Comment:
Do the global lexicals get optimized by the 8-magic-lines optimization mechanism into lexical constants? Would be nice but not crucial. Mostly I'm just curious if you happen to know the answer.

Reply:
The globalLexicals do get optimized by the evaluator.

Reply:
> The globalLexicals do get optimized by the evaluator.

Excellent!